catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/features/CamelFeatures.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.content.monitor.features;
+
+import static org.ops4j.pax.exam.CoreOptions.maven;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.features;
+
+import java.util.Arrays;
+
+import org.ops4j.pax.exam.Option;
+
+public class CamelFeatures {
+    public enum CamelFeature {
+        XML_SPECS_API("xml-specs-api"), CAMEL("camel"), CAMEL_CORE("camel-core"), CAMEL_CATALOG(
```

Comment:
The enums were generated out of the camel feature file. I thought it would be nice to have all features of camel available if we needed them in other component tests. Reasoning for an enum over a string array was for compile time checks to make sure we were not trying to start feature that wasn't available in the camel feature file.